examples/callback.py

```diff
@@ -55,6 +56,14 @@ def stop(server, callback):
         server, callback = loop.run_until_complete(start())
         # Ctrl+C: Terminate the server and the callback application
         try:
+            print('Listening on:\n')
+            for socket in server.sockets:
+                host = socket.getsockname()[0]
+                port = socket.getsockname()[1]
+                if host == '::':
+                    host = '[::]'
+                print('  https://{}:{}{}'.format(host, port, callback.route))
```

Comment:
> Did you forget about tuple unpacking

Whoops, yes :) Although the `*_` is not necessary, right?

Nice, I didn't know the `ipaddress` package yet. It doesn't seem to be able to handle `::` though:

```
>>> ipaddress.ip_address('::')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/dist-packages/ipaddress.py", line 163, in ip_address
    ' a unicode object?' % address)
ipaddress.AddressValueError: '::' does not appear to be an IPv4 or IPv6 address. Did you pass in a bytes (str in Python 2) instead of a unicode object?
```

Reply:
> Whoops, yes :) Although the *_ is not necessary, right?

It's necessary because [you get a four-tuple for IPv6](https://docs.python.org/3/library/socket.html#socket-families)... which you would need to parse as well if you wanted to be 100% correct.

Works in Python >= 3.4 which is our base requirement.

```
Python 3.4.3 (default, Nov 28 2017, 16:41:13) 
[GCC 4.8.4] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import ipaddress
>>> ipaddress.ip_address('::')
IPv6Address('::')
```

Reply:
FYI:
* flow info: https://tools.ietf.org/html/rfc6437
* scope id: https://en.wikipedia.org/wiki/IPv6_address#Scoped_literal_IPv6_addresses